src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -86,6 +86,38 @@ SentrySDK.start { options in
 [NSURLSession]: https://developer.apple.com/documentation/foundation/nsurlsession
 [NSURLConnection]: https://developer.apple.com/documentation/foundation/nsurlconnection
 
+## File I/O Instrumentation
+
+<Alert level="info" title="Important">
+
+This is an experimental feature and requires an opt-in.
+
+</Alert>
+
+The Sentry SDK adds spans for file I/O operations to ongoing transactions bound to the Scope. Currently, the SDK supports operations with [NSData][NSData], but many other Apple APIs like [NSFileManager][NSFileManager] uses [NSData][NSData].
```

Comment:
`m`: Can we list the Apple APIs we know that use NSData internally, please?

Reply:
Well, there is a ton of APIs that use NSData internally. 
But if you mean APIs available to load data directly I could think of NSFileManager, NSString and NSBundle.
I would add this as examples, but I don't believe a complete list is relevant here.